python/lsst/ts/externalscripts/auxtel/make_latiss_calibrations.py

```diff
@@ -86,6 +86,17 @@ def get_schema(cls):
         description: Configuration for making a LATISS calibrations SAL Script.
         type: object
         properties:
+            detectors:
+                description: Detector IDs.
+                type: array
+                items:
+                  - type: number
```

Comment:
number is either int or float... I think detector numbers are only integer no?

Reply:
Yes. I had ``integer`` before in there because of the reason you cite, but that was breaking the schema. Should it be ``int`` instead?

Reply:
should be `integer`: https://json-schema.org/understanding-json-schema/reference/numeric.html#id4